domain-cc/cc-app/src/python_src/util/lookup_table.py

```diff
@@ -2,27 +2,47 @@
 import os
 from abc import ABC
 
-from .condition_dropdown_table_version import CONDITION_DROPDOWN_TABLE_VERSION
-from .diagnostic_code_table_version import TABLE_VERSION
+from .table_versions import (
+    CONDITION_DROPDOWN_TABLE_VERSION,
+    DIAGNOSITC_CODE_TABLE_VERSION,
+)
 
 # https://docs.google.com/spreadsheets/d/18Mwnn9-cvJIRRupQyQ2zLYOBm3bd0pr4kKlsZtFiyc0/edit#gid=1711756762
-dc_table_name = f"Contention Classification Diagnostic Codes Lookup table master sheet - DC Lookup {TABLE_VERSION}.csv"
+dc_table_name = (
+    f"Contention Classification Diagnostic Codes Lookup table master"
+    f" sheet - DC Lookup {DIAGNOSITC_CODE_TABLE_VERSION}.csv"
+)
 # https://docs.google.com/spreadsheets/d/1A5JuYwn39mHE5Mk1HazN-mxCL2TENPeyUPHHhH10g_I/edit#gid=819850041
-condition_dropdown_table_name = f"Contention dropdown to classification master - Dropdown Lookup {CONDITION_DROPDOWN_TABLE_VERSION}.csv"
+previous_condition_dropdown_table_name = (
+    "Contention dropdown to classification master - Dropdown Lookup v0.1.csv"
+)
+
+condition_dropdown_table_name = (
+    "Contention dropdown to classification master"
+    f" - Prototype - Terms + mapping {CONDITION_DROPDOWN_TABLE_VERSION}.csv"
+)
 
 
 class LookupTable(ABC):
-    """Generalized lookup table for mapping input strings to contention classification codes"""
+    """
+    Generalized lookup table for mapping input strings to contention
+    classification codes
+    """
 
     CSV_FILEPATH = None
+    REDESIGNED_FILEPATH = None
```

Comment:
:pick: why not just describe which csv if there will be an import for each one? e.g. `V2_FILEPATH`??